src/components/filtered-list/filtered-list-item/filtered-list-item.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./filtered-list-item.styles.scss";
+import CaretSvg from "../../../images/icons/caret.svg";
+
+export class FilteredListItem extends React.Component {
```

Comment:
All components in here are function components. For the sake of consistency, could you please convert yours to function components as well?

Reply:
Refactored FilteredListItem to use a function. 
@surdu For FilteredList I need to have local state, do you recommend a different approach?